README.md

```diff
@@ -1,15 +1,162 @@
+
+## This Repository
+
+This is the mono-repository for [Quire](https://quire.getty.edu/). It contains:
+
+-   [packages/cli](https://github.com/thegetty/quire/tree/main/packages): the quire command-line interface
+-   [starters/default](https://github.com/thegetty/quire/tree/main/starters): a default starter project
+-   [themes/default](https://github.com/thegetty/quire/tree/main/themes): the default publication them
+
+
+## Installing the Quire CLI
+
+### Quick Install: MacOS
+
+1.  Install Apple’s Xcode with: `xcode-select --install`
+2.  Install the LTS version of Node.js: https://nodejs.org
+3.  Install Quire with: `npm install --global \@thegetty/quire-cli`
+4.  Confirm by pulling up a list of Quire commands: `quire --help`
+
+### Quick Install: Windows
+
+1.  Install Git for Windows: https://gitforwindows.org/
+2.  Install the LTS version of Node.js: https://nodejs.org
+3.  Install Windows Build Tools with: `npm install --g --production windows-build-tools`
+4.  Install Quire with: `npm install --global \@thegetty/quire-cli`
+5.  Confirm by pulling up a list of Quire commands: `quire --help`
+
+For full installation instructions, and information about updating and uninstalling Quire please see our [documentation.](https://quire.netlify.app/documentation/install-uninstall/)
+
+### Quire CLI Commands 
+
+
+| Option          | Result                                  | 
+| --------------- | --------------------------------------- | 
+| -V, --version   | Output the version number               |
+| -v, --verbose   | Log verbose output                      |
+| -f, --file      | Add Filename and optional new filepath  |
+| -e, --env       | Add environment variable                |
+| -h, --help      | Output usage information                |
+
+
+| Command             | Result                                                            |
+| ------------------- | ----------------------------------------------------------------- |
+| new \<projectName\> | Create a new Quire project in the current directory.              |
+| preview             | Run the preview server in the current directory                   |
+| install             | Install this project's theme dependencies                         |
+| site                | Run the build command in the current directory                    |
+| pdf                 | Generate a PDF version of the current project                     |
+| epub                | Generate an EPUB version of the current project                   |
+| mobi                | Generate an MOBI version of the current project                   |
+| template            | Download templates to customize your file output (only for EPUB)  |
+| debug               | Development use only - log info about current  project            |
+
+For more commands please see the [*Quire Commands*](https://quire.getty.edu/documentation/quire-commands/) section of our [documentation](https://quire.getty.edu/documentation/quire-commands/) 
```

Comment:
Maybe add "about these" as in "For more about these commands ..."? I don't think we have more commands otherwise?